SwanProjects/bin/swan_kmspecs

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+import argparse
+import json
+import os
+import pprint
+import subprocess
+import sys
+from distutils.spawn import find_executable
+from shutil import rmtree
+
+def _checkipykernel(project_path,python_interpreter,python_version):
```

Comment:
Please add docstrings for every function, explaining what they do, what their parameters and return value are. You can use e.g. the Numpy style: https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
The file allows to get the information such if ipykernel is installed in the stack and the kernels paths for our kernel manager.

I put it in the documentation.